Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -338,23 +340,31 @@ type HumanReadablePrinter struct {
 }
 
 // NewHumanReadablePrinter creates a HumanReadablePrinter.
-func NewHumanReadablePrinter(noHeaders, withNamespace bool, wide bool, showAll bool, showLabels bool, absoluteTimestamps bool, columnLabels []string) *HumanReadablePrinter {
+func NewHumanReadablePrinter(printOptions *PrintOptions) *HumanReadablePrinter {
```

Comment:
If all we do with `printOptions` is an indirection, `options : *printOptions`, why do we make the argument be a pointer type?